re.sonny.Workbench.Devel.json

```diff
@@ -148,6 +99,11 @@
           "type": "dir",
           "path": "."
         }
+      ],
+      "post-install": [
+        "cp -a /usr/lib/sdk/vala/bin/* /app/bin/",
```

Comment:
> cp: cannot stat '/usr/lib/sdk/vala/bin/*': No such file or directory

can you use `install` ?

Reply:
This is strange. it seems like the vala sdk is not mounted into the extension point??

Reply:
This is a flatpak-vscode only - I fixed it in this commit https://github.com/sonnyp/Workbench/pull/273/commits/e05ff73b08cdd15e38b05d5e66f14a48c7d6016f